vars/helmInstall.groovy

```diff
@@ -54,6 +55,26 @@ def call(DockerImage dockerImage, Map params) {
     // default values + overrides
     def templateValues = "${helmResourcesDir}/${chartName}/values.template.yaml"
     def defaultValues = "${helmResourcesDir}/${chartName}/values.yaml"
+    if (fileExists(defaultValues)) {
+      onPR{
+        aksEnvironment ="preview"
```

Comment:
we should probably use this instead of hardcoding the name?

https://github.com/hmcts/cnp-jenkins-library/blob/master/src/uk/gov/hmcts/contino/Environment.groovy#L7